app/models/user.rb

```diff
@@ -21,7 +23,7 @@ def login
 
   def self.find_for_database_authentication(warden_conditions)
     conditions = warden_conditions.dup
-    if login = conditions.delete(:login)
+    if (login = conditions.delete(:login))
```

Comment:
why do you add brackets?

Reply:
RubyMine suggested add clarifying brackets and I did it